cli/ingress-controller/flags.go

```diff
@@ -158,6 +160,7 @@ The controller will set the endpoint records on the ingress using this address.`
 		UpdateStatus:           *updateStatus,
 		ElectionID:             *electionID,
 		EnableProfiling:        *profiling,
+		KongOutsideKubernetes:  *kongOutsideKubernetes,
```

Comment:
What do you think of calling this flag `EnableExternalSupport`, since the concept is supporting an external gateway? We could then also refactor the methods to be more aligned with the concept, such as `getExternalEndpoints`.